src/Microsoft.Extensions.Configuration.UserSecrets/build/netstandard1.0/Microsoft.Extensions.Configuration.UserSecrets.props

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(UserSecretsAssemblyAttributeFile)' == '' and $(MSBuildProjectFile.EndsWith('.vbproj')">
```

Comment:
I think I have tried to use that task before and had some kind of issue, but if it's working for you, definitely use it :).